tests/feature_selection/test_selection.py

```diff
@@ -3,28 +3,54 @@
 # Maximilian Christ (maximilianchrist.com), Blue Yonder Gmbh, 2016
 
 import numpy as np
-from unittest import TestCase
 import pandas as pd
+from pytest import raises
+from future.utils import lrange
 
 from tsfresh.feature_selection.selection import select_features
 
 
-class SelectionTestCase(TestCase):
+def test_assert_list():
+    with raises(TypeError):
```

Comment:
Nothing. I just prefer the tests to be 'flat', i.e. not part of a class here, as the tests don't share any data.